test/solidity/codegenTests/namedCall.sol

```diff
@@ -1,16 +1,15 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-// REQUIRES: namedcall
```

Comment:
Don't remove the `REQUIRES` statement. Instead, you should add `namedcall` in the `config.available_features`. See Line 27 in the `test/lit.cfg.py`